src/SFTPClient.cpp

```diff
@@ -207,6 +206,11 @@ int SFTPClient::disconnectUnlocked(bool force, int timeout_ms, AbstractDisconnec
    return rc;
 }
 
+int SFTPClient::sftpDisconnect(bool force, int timeout_ms, AbstractDisconnectionHelper* adh, ExceptionSink* xsink) {
```

Comment:
this function is not necessary as per the comments on SFTPClient::disconnect() above